task/buildah-oci-ta/0.2/buildah-oci-ta.yaml

```diff
@@ -475,96 +475,24 @@ spec:
           memory: 256Mi
       securityContext:
         runAsUser: 0
-    - name: merge-syft-sboms
-      image: registry.access.redhat.com/ubi9/python-39:1-192.1722518946@sha256:0176b477075984d5a502253f951d2502f0763c551275f9585ac515b9f241d73d
+    - name: prepare-sboms
+      image: quay.io/redhat-appstudio/sbom-utility-scripts-image@sha256:064eeec1cfb75abd21b533bf0d9870a7a3fb138f30192596dce28061798cbf06
```

Comment:
I'm not sure we need `-image` suffix in the mage name.

Reply:
We don't, but not sure if it is worth it renaming it now. I would have to create a new repo in quay, create additional PR in build-task-dockerfiles and update it in this one as well.